sdk/textanalytics/ai-text-analytics/CHANGELOG.md

```diff
@@ -1,5 +1,18 @@
 # Release History
 
+## 1.0.0-preview.2 (2020-02-11)
+
+- Added support for rotating API keys via the `TextAnalyticsApiKeyCredential.updateKey` method. #6972
+- Added a discriminant type for `TextAnalyticsResult.error` to allow for easy differentiation between success and error types using `if (result.error) { ... }` in both TypeScript and JavaScript. #7046
+- [Breaking] Removed the `Entity` type and added new return types `CategorizedEntity` and `PiiEntity` for the `recognizeEntities` and `recognizePiiEntities` methods respectively. These types are equivalent to the previous `Entity` type. #7220
+- [Breaking] Removed the `detectedLanguages` property from `DetectLanguageResult`, as the service currently only returns one language. #7220
+- [Breaking] Renamed the `detectLanguages` method to `detectLanguage`, as the `detectedLanguages` property was removed from the `DetectLanguageResult` return type. #7220
```

Comment:
I meant

Renamed the `detectLanguages` method of **SomeClient** to `detectLanguage` 

but it's minor.

Reply:
Ah I see. We only have one client class in TA, so I am not too concerned about this.